src/App.css

```diff
@@ -1,5 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Londrina+Solid:wght@900&display=swap');
 .App {
     width: 60vw;
     margin: 0 auto;
 }
 
+body::-webkit-scrollbar {
+    width: 1em;
+}
+   
+body::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 2px rgba(0, 0, 0, 0.3);
+}
+   
+body::-webkit-scrollbar-thumb {
+    background-image: url("https://img.freepik.com/free-vector/yellow-toy-block_1284-42372.jpg?w=360");
+    background-repeat: repeat-y;
+    background-size: 150px;
```

Comment:
This is very cool! I learned many new things from this!
Quick question, does this work on non-webkit browsers (e.g., Firefox) or do you need to include the equivalent properties for them? That's what MDN seems to suggest but I don't have firsthand experience with this.

Reply:
Great question, you're right - the scrollbar css using webkit doesn't work in Firefox! I wasn't able to find any way to add a url image to the scrollbar for Firefox, but I just found a way to change the color and width of the scrollbar that is compatible with Firefox.